browser/main/Detail/index.js

```diff
@@ -14,7 +14,12 @@ class Detail extends React.Component {
     super(props)
 
     this.focusHandler = () => {
+      if (this.refs.root.state.status === 'CODE') return
       this.refs.root != null && this.refs.root.focus()
+      let { config } = this.props
+      if (config.editor.switchPreview === 'RIGHTCLICK') {
+        this.refs.root.setState({status: ''})
```

Comment:
[NOT MUST] It is not good way to use React component.
Can you keep `status` state on this component and pass it as props to `this.refs.root`?